powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationServiceBehavior.java

```diff
@@ -1534,7 +1540,15 @@ public CommitActivationResponse commitActivation(CommitActivationRequest request
             }
 
             // Validate activation OTP
-            validateActivationOtp(com.wultra.security.powerauth.client.model.enumeration.ActivationOtpValidation.ON_COMMIT, activationOtp, activation, externalUserId);
+            validateActivationOtp(activationOtp, activation, externalUserId);
```

Comment:
empty activationOtp is validated as valid (similar to in the previous phase).

Reply:
See:
https://github.com/wultra/powerauth-server/pull/1671/commits/495f1302182cbd04bfe58ea043d92df0b8c8583d